### PR TITLE
8607 update sub text

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8605)

This is one of a few rounds of content updates that need to be made to the 686 before launch. 

## Tasks

1. We need to add the breadcrumbs component to the 686. We assume this is a pre-built component and just needs to be configured somewhere but some research will need to be done as to how these are added to a form.
2. Add the 'need help' section to the bottom of each page of the 686. We assume this is a pre-built component, since it exists on so many other forms, and we just need to configure it somewhere in the form but some research will need to be done as to how this is added to the form.
3. Add in-form text to match mockups. The text VA Form 21-686c (and 21-674) needs to be added into the form where shown in the mockups ( a link to the mockups is provided through the list linked above ). This text may be something that can just be configured in the form system OR it may need to be a custom component we build. Some research will need to be done for this.

## Screenshots

Screen Shot 2020-05-14 at 10 10 28 AM
Screen Shot 2020-05-14 at 10 10 37 AM
Screen Shot 2020-05-14 at 10 10 52 AM

## Acceptance criteria
- [x] Breadcrumbs have been added to the form
- [x] A 'need help' section have been added to the bottom of all pages of the 686 form
- [x] Needed text has been added to the form 686

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/components/GetFormHelp.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const GetFormHelp = () => (
+  <p className="help-talk">
+    Need help filling out the form or have questions about eligibility? Please
+    call VA Benefits and Services at{' '}
+    <a className="nowrap" href="tel:1-800-827-1000">
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/components/GetFormHelp.jsx:7]: This should include an aria-label with the value `1. 800. 8 2 7. 1000.`

File: src/applications/registry.json [hunk 2]
```diff
@@ -262,12 +262,22 @@
     }
   },
   {
-    "appName": "New 686",
+    "appName": "686c-674",
     "entryName": "new-686",
```
Comment (micahchiang) [inline src/applications/registry.json:266]: I wonder if `entryName` should change too. Might be worth changing this to mirror the appName.

File: src/applications/disability-benefits/686c-674/manifest.json [hunk 3]
```diff
@@ -1,7 +1,10 @@
 {
-  "appName": "New 686",
+  "appName": "686c-674",
   "entryFile": "./app-entry.jsx",
   "entryName": "new-686",
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/manifest.json:4]: See comment below regarding entryName.

**Review by micahchiang — APPROVED** [review]
Approved, with a suggestion to add an aria-label to the phone number link, as well as another thought about entryName.

File: src/applications/disability-benefits/686c-674/components/GetFormHelp.jsx (hunk 1 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/components/GetFormHelp.jsx:7]: Added this

File: src/applications/disability-benefits/686c-674/manifest.json (hunk 3 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/manifest.json:4]: Changed this

File: src/applications/registry.json (hunk 2 above)
Comment (cohnjesse) [reply src/applications/registry.json:266]: Changed this

File: src/applications/registry.json [hunk 4]
```diff
@@ -262,12 +262,22 @@
...
-      "vagovprod": false
+      "vagovprod": false,
+      "includeBreadcrumbs": true,
+      "breadcrumbs_override": [
+        {
+          "path": "/view-change-dependents/view",
+          "name": "View or change your dependents"
+        },
+        {
+          "name": "686c-674"
```
Comment (Mottie) [inline src/applications/registry.json:278]: I think the common practice is to add the name of the form here, not the number.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**